packages/xrpl/src/models/methods/nftsByIssuer.ts

```diff
@@ -0,0 +1,68 @@
+import { NFToken } from '../common'
+
+import { BaseRequest, BaseResponse, LookupByLedgerRequest } from './baseMethod'
+
+/**
+ * The nfts_by_issuer method returns a list of NFTokens issued by the account.
+ * The order of the NFTs is not associated with the date the NFTs were minted.
+ * Expects a response in the form of a {@link
+ * NFTsByIssuerResponse}.
+ *
+ * @category Requests
+ */
+export interface NFTsByIssuerRequest
+  extends BaseRequest,
+    LookupByLedgerRequest {
+  method: 'nfts_by_issuer'
```

Comment:
Should be `command` instead of `method` since JS library deals with websocket client

Reply:
I followed https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer/ , it's weird that only this page the field is "method" instead of "command". I can change it to "command" for sure.

Reply:
`command` is used when the connection is websocket, eg. your JS script opens a websocket connection. Whereas `method` field is used when the connection is http (eg. curl command)

For example, you could use this in the cmdline to query for the API,
`curl 127.0.0.1:51233  -d '{"method":"nfts_by_issuer", "params": [{"issuer": "rfXeQv31yWMrhhPxMHZRzQqhw5mQrcuici"}]}'`